src/syl.cpp

```diff
@@ -192,4 +193,48 @@ int Syl::ResetDrawing(FunctorParams *functorParams)
     return TimeSpanningInterface::InterfaceResetDrawing(functorParams, this);
 }
 
+bool Syl::CreateDefaultZone(Doc *doc)
+{
+    Zone *zone = new Zone();
+    const int offsetUly = 100;
+    const int offsetLrx = 100;
+    const int offsetLry = 200;
+
+    LayerElement *syllable = dynamic_cast<LayerElement *>(this->GetFirstAncestor(SYLLABLE));
+    assert(syllable);
```

Comment:
This is problematic because with non neume notation there will not be a Syllable parent. Just check and return instead of assert it.